tools/pydocs/src/conf.py

```diff
@@ -0,0 +1,132 @@
+# This file is execfile()d with the current directory set to its
```

Comment:
I added a comment that it is based on a file generated by `sphinx-autogen`.  (Also added a copyright line, since I forgot that earlier.)